pkg/util/log/log.go

```diff
@@ -40,7 +40,7 @@ func InitLogger(cfg *server.Config, reg prometheus.Registerer, buffered bool, sy
 	// this works in go 1.12, but doesn't work in versions earlier.
 	// it will always shows the wrapper function generated by compiler
 	// marked <autogenerated> in old versions.
-	cfg.Log = dslog.GoKit(log.With(l, "caller", log.Caller(4)))
+	cfg.Log = log.With(l, "caller", log.Caller(4))
```

Comment:
I wonder if `log.Caller(4)` is correct now that we've removed the outer function call. Can you check pls?

Reply:
Will try to find out @dannykopping.

Reply:
@dannykopping you're right, 3 levels is now enough (I tested), so I just changed the code into setting `cfg.Log = Logger`, since they should be the same.